cmd/lambda/leases/get.go

```diff
@@ -49,14 +49,14 @@ func GetLeasesByPrincipcalIDAndAccountID(w http.ResponseWriter, r *http.Request)
 		response.WriteServerErrorWithResponse(w, errMsg)
 		return
 	}
-	if lease == nil {
-		log.Printf("Error Getting Lease for Id: %s", err)
-		response.WriteNotFoundError(w)
-		return
+	leaseResponses := []*response.LeaseResponse{}
```

Comment:
So one object was coming back but now an array is coming back. Does that break the contract?

Reply:
Yes. Though I should point out that at one point we _were_ returning empty lists instead of a 404, (as of v0.23.0, at least). So we more recently broke the contract, this change is putting it back.

Reply:
sorry, I got confused here. The API that broke at some point was the `GET /leases?principalId=`
we'd always had `GET /leases?principalId=&accountId=` return a single object (or 404),
as legacy behavior from before we had Lease IDs.

But I would like to change that, and I'm ok calling that a breaking change to our API.